core/domain/question_fetchers.py

```diff
@@ -31,29 +31,26 @@
 
 
 def get_questions_and_skill_descriptions_by_skill_ids(
-        question_count, skill_ids, start_cursor):
+        question_count, skill_ids, offset=0):
```

Comment:
Make the offset mandatory and do not include "offset=" when calling this function.

Reply:
Done